### PR TITLE
fix #81240 ppss.kr

### LINKED ISSUE #81240: ppss.kr
### Issue URL (Ads)
 [https://ppss.kr/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fppss.kr%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.34
Browser: | Firefox
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (2880), <br>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion<br/><br/><b>Language-specific:</b><br/> List-KR

### PR DESCRIPTION
**Pull request by stanislav-atr**
#81240

### CODE REVIEW

File: EnglishFilter/sections/foreign.txt [hunk 1]
```diff
@@ -5616,6 +5616,8 @@ g-enews.com##.mad
 ! https://github.com/AdguardTeam/AdguardFilters/issues/80447
 ||data.news.zumst.com/alyac/popular/news.json
 121.189.38.123^$network
+! https://github.com/AdguardTeam/AdguardFilters/issues/81240
+ppss.kr#%#//scriptlet('abort-on-property-write', 'adshield_recovery_entrypoint')
```
Comment (Sergey-Lyapin) [inline EnglishFilter/sections/foreign.txt:5620]: What about platforms without scriptlets support?

Comment (zloyden) [reply EnglishFilter/sections/foreign.txt:5620]: @stanislav-atr try these rules
```
@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=ppss.kr
@@||ad.doubleclick.net/ddm/$xmlhttprequest,domain=ppss.kr
ppss.kr#@#.adsbygoogle
ppss.kr#@##ad-unit
ppss.kr#@#.ad-unit
```

Comment (stanislav-atr) [reply EnglishFilter/sections/foreign.txt:5620]: @zloyden, works fine on my end.

**Review by zloyden — APPROVED** [review]

**Comment by JobcenterTycoon** [comment]
ads are back

**Pull request merged**